Signal/src/Models/TSMessageAdapaters/TSAnimatedAdapter.m

```diff
@@ -15,13 +15,12 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @interface TSAnimatedAdapter ()
-//<FLAnimatedImageViewDebugDelegate>
 
 @property (nonatomic, nullable) FLAnimatedImageView *cachedImageView;
-@property (nonatomic) UIImage *image;
```

Comment:
We shouldn't load and retain animated GIFs in memory until we're trying to present them.